pkg/seedadmissioncontroller/webhooks/admission/extensionresources/templates_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extensionresources_test
+
+var (
+	backupBucketFmt = `
+{ "apiVersion": "extensions.gardener.cloud/v1alpha1",
+"kind": "BackupBucket",
+"metadata": {
+  "name": "cloud--gcp--fg2d6",
+  %s
+  "namespace": "prjswebhooks"
+  },
+"spec":{
+  "type": %q,
+  "region": "eu-west-1",
+  "secretRef": {
+    "name": %q,
+    "namespace": "garden"
+  }
+}
+}
```

Comment:
Isn't better to have a typed go object instead of such string templates (that can easily get outdated/invalid over time)?